lektor/cli.py

```diff
@@ -251,11 +251,13 @@ def clean_cmd(ctx, output_path, verbosity):
 @click.option('--key', envvar='LEKTOR_DEPLOY_KEY',
               help='The contents of a key file directly a string that should '
               'be used for authentication of the deployment.')
+@click.option('--message', default=None,
+               help='Add custom commit message upon deployment')
```

Comment:
This help message talks specifically about a commit message, but it's conceivable that you could use this flag to provide a message for something other than a commit. Can you make this a bit more generic? It would be up to the Publisher object how to use this value, if at all.